jquery.lazyload.js

```diff
@@ -126,6 +126,9 @@
                                 var elements_left = elements.length;
                                 settings.load.call(self, elements_left, settings);
                             }
+                        }).bind("error", function() {
+                            var fallback = $self.attr("data-fallback");
+                            $self.attr("src", fallback);
```

Comment:
:+1: It would be easy to implement: `var fallback = $self.attr("data-fallback") || settings.placeholder;`